build.gradle

```diff
@@ -101,6 +100,10 @@ allprojects {
             url "https://maven.research.rackspacecloud.com/content/groups/aggregate/"
         }
     }
+
+    ext {
+        docsHome = "http://www.openrepose.org/versions/$version/index.html".toString()
+    }
```

Comment:
I originally had it above, but version is not yet defined at that point. And for other reasons version can't by in the outer scope, so it has to be down here.